include/HL_sci.h

```diff
@@ -138,7 +138,7 @@ typedef struct sci_config_reg
                                    |(uint32)((uint32)0U << 0U))
                                    
 #define SCI1_FORMAT_CONFIGVALUE     (8U - 1U)
-#define SCI1_BRS_CONFIGVALUE        (487U)
+#define SCI1_BRS_CONFIGVALUE        (40U)

```

Comment:
What are these changes?

Reply:
Those look like they are hal generated. We did make some changes to the hal files. The registers and speeds were not set correctly for certain lines.

Reply:
Ah. This is because we changed the UART baudrate. Won't affect anything else.